english/eating-walls/find-a-toilet/door/bear/run-into-room.md

```diff
@@ -0,0 +1,7 @@
+You run into the room and slam the door behind you! You don't know whose hand that was and you don't want to.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.